startup.sh

```diff
@@ -79,6 +83,8 @@ npm install -g yo generator-rn-toolbox
 yo rn-toolbox:assets --icon ../lib/react-native/icon.png
 mv ../lib/react-native/fernet.js node_modules/fernet/fernet.js
 cd ..
-rm -rf .git lib server startup.sh
-echo "\n\n${purple}Everything is setup! You can now run `fbs run` or `react-native run-(ios|android)`${normal}"
-echo "${purple}To learn more about usage and building for the app, go to https://build-system.fman.io/ and https://facebook.github.io/react-native/${normal}\n\n"
\ No newline at end of file
+
+### Clean up
+#rm -rf .git lib server startup.sh
```

Comment:
This has been removed for now as we discuss the details of expectations for other developers who use this project.